app/models/match_request.rb

```diff
@@ -36,9 +36,22 @@ class MatchRequest < ApplicationRecord
 
   validates_with OwnRequestValidator
   validates :response_datetime, presence: true, unless: :status_is_pending?
+  validates :status, presence: true
 
   def status_is_pending?
     return status == :pending.to_s
   end
 
+  def status_is_cancelled?
+    return status == :cancelled.to_s
+  end
+
+  def status_is_accepted?
+    return status == :accepted.to_s
+  end
+
+  def status_is_refused?
+    return status == :refused.to_s
+  end
```

Comment:
I actually wasn't aware of this, but I created these methods in case I want to change the names in the enum in the future, then I would only have to change in the method instead of in every situtation where I call it